### PR TITLE
Feat/migrate-to-polkadot-sdk-repo

### PR DESCRIPTION
**Pull request by dandanlen**
# Pull Request

Closes: PRO-890

## Checklist

Please conduct a thorough self-review before opening the PR.

- [x] I am confident that the code works.
- [x] I have updated documentation where appropriate.

## Summary

This migrates us to the polkadot-sdk repo now that the substrate repo is no longer maintained. 

There is no exact correspondence between tags in the old and new repos so some minor changes to imports were required. 

We now build against polkadot tags. In this case, a slightly modified polkadot-1.1.

This will make it easier to upgrade the dependencies in the future.

### CODE REVIEW

File: state-chain/node/chainspecs/berghain.chainspec.raw.json
Comment (AlastairHolmes) [inline state-chain/node/chainspecs/berghain.chainspec.raw.json:6]: ?

**Review by AlastairHolmes — APPROVED** [review]

File: state-chain/node/src/command.rs
Comment (dandanlen) [reply state-chain/node/src/command.rs:193]: You're right, better to remove it.

**Comment by dandanlen** [comment]
Fixed those issues and rebased against main. 

I'll wait until #4382 is merged and will rebase again after that (since there's a new pallet in that PR) before merging this to main.

**Pull request merged**
